overlord/configstate/configcore/vitality.go

```diff
@@ -166,9 +166,15 @@ func handleVitalityConfiguration(tr RunTransaction, opts *fsOnlyContext) error {
 		if err != nil {
 			return err
 		}
-		flags := &wrappers.StartServicesFlags{Enable: true}
+
+		// TODO: add user-daemons this once QueryDisabledServices
```

Comment:
Just thinking, once we start querying disabled services for each user, this will become slightly messy, as we'll first have an outgoing call to collect this, aggregate it here and then do a call again for each user. Could we instead leave this choice to the session agent to do the query locally and act on the results when handling "start" action? If we really want, the response could contain the result with a list of services started and disabled.

Reply:
This is a good idea, i'll look into this